src/router/index.js

```diff
@@ -26,6 +27,16 @@ const routes = [
     name: "Users Details",
     component: UserDetails,
     meta: { title: "Users Details" },
+    // children: [
+    //   { path: "add", component: ResourceDetails },
+    // ]
+  },
+  {
+    path: "/users/:id/add",
```

Comment:
And perhaps we can reuse the component for removing quantity and it will be on `/resources/availability/remove/{resourceId}`. Let's discuss it offline.